deploy/gke/redaction/README.md

```diff
@@ -68,4 +68,105 @@ When you are done, you can clean up everything you've done with the following st
 
 ```
 kubectl delete namespace $OTEL_NAMESPACE
-```
\ No newline at end of file
+```
+
+## Try it out yourself (Optional)
```

Comment:
This should go before the cleanup section.  (if someone cleans up, then the try it out won't work).